testnet/stacks-node/src/config.rs

```diff
@@ -494,11 +496,97 @@ impl Config {
                     rbf_fee_increment: burnchain
                         .rbf_fee_increment
                         .unwrap_or(default_burnchain_config.rbf_fee_increment),
-                    epochs: match burnchain.epochs {
-                        Some(epochs) => Some(epochs),
-                        None => default_burnchain_config.epochs,
-                    },
+                    epochs: default_burnchain_config.epochs,
+                };
+
+                if let Some(ref conf_epochs) = burnchain.epochs {
+                    let default_epochs = match result.get_bitcoin_network().1 {
+                        BitcoinNetworkType::Mainnet => {
+                            panic!("Cannot configure epochs in mainnet mode");
```

Comment:
Ideally we'd return an Err here.

Reply:
Addressed in https://github.com/stacks-network/stacks-blockchain/pull/3207/commits/0f4ac0655a567b830b6cfe2818378e22f3b51f0e